packages/ui/package.json

```diff
@@ -16,11 +16,12 @@
   ],
   "scripts": {
     "build:readme": "cp ./README.md ./build/README.md",
-    "build:copy-files": "cp ../../(LICENSE|NOTICE) ./build/",
+    "build:copy-files": "cp ../../(LICENSE|NOTICE) ./build/ && cp ./LICENSE-THIRD-PARTY ./build/",
     "build:package-json": "(cat package.json | jq 'del(.devDependencies, .resolutions, .scripts, .\"lint-staged\")') > ./build/package.json",
     "build:package": "tsc --build tsconfig.build.json",
-    "build": "yarn build:package && yarn build:package-json && yarn build:readme && yarn build:copy-files",
+    "build": "yarn build:package && yarn generate:attribution && yarn build:package-json && yarn build:readme && yarn build:copy-files",
     "check:all": "yarn test:ci && yarn build && yarn storybook:build && yarn styleguide:build",
+    "generate:attribution": "generate-attribution && mv oss-attribution/attribution.txt LICENSE-THIRD-PARTY",
```

Comment:
should this be `yarn run generate-attribution`?

Reply:
Thanks. Usually yarn has the bin folder in its path. But I have add the a yarn to avoid confusion.